src/Resources/translations/CmfSonataAdminIntegrationBundle.it.xliff

```diff
@@ -286,14 +284,12 @@
             <!-- $Core -->
             <trans-unit id="form.tab_publish">
                 <source>form.tab_publish</source>
-                <!-- TODO: Translate -->
-                <target>Publish Settings</target>
+                <target>Pubblicazione</target>
             </trans-unit>
 
             <trans-unit id="form.group_publish_workflow">
                 <source>form.group_publish_workflow</source>
-                <!-- TODO: Translate -->
-                <target>Status</target>
+                <target>Workflow pubblicazione</target>
```

Comment:
Instead of the original "Publish Workflow", I decided to name this "Status" (as in: In what publication state is this document?). Is there maybe a better fitting spanish translation for this?